msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.ObjCBinding.CSharp.targets

```diff
@@ -28,6 +28,11 @@ Copyright (C) 2014 Xamarin Inc. All rights reserved.
 	<Import Project="Xamarin.Mac.ObjCBinding.Common.targets" />
 
 	<PropertyGroup>
+		<IsXBuild Condition="'$(MSBuildRuntimeVersion)' == ''">true</IsXBuild>
```

Comment:
So I tested it out and with msbuild fsharp works with this change, I did not dig into why.

Now, that file in particular is with binding project, but they really don't have F# support (you can bind the libs and use them in F# though).